### PR TITLE
PMM-7326 log level for MySQL, MongoDB, PostgreSQL, ProxySQL exporters

### PR DESCRIPTION
**Pull request by YaroslavPodorvanov**
PMM-7326

Build: SUBMODULES-2471

- [ ] https://github.com/percona/pmm/pull/874
- [ ] https://github.com/percona/pmm-admin/pull/202
- [ ] https://github.com/percona/pmm-managed/pull/1087

### CODE REVIEW

**Comment by YaroslavPodorvanov** [comment]
@ritbl, @BupycHuk, @qwest812 can you review?

File: models/agent_helpers.go [hunk 1]
```diff
@@ -827,6 +828,7 @@ func CreateAgent(q *reform.Querier, agentType AgentType, params *CreateAgentPara
 		AzureOptions:                   params.AzureOptions,
 		PushMetrics:                    params.PushMetrics,
 		DisabledCollectors:             params.DisableCollectors,
+		LogLevel:                       &params.LogLevel,
```
Comment (BupycHuk) [inline models/agent_helpers.go:831]: it will make LogLevel allways not nil. let's use `pointer.ToStringOrNil`

**Review by BupycHuk — APPROVED** [review]
Approved, but need some fix

File: models/agent_helpers.go (hunk 1 above)
Comment (YaroslavPodorvanov) [reply models/agent_helpers.go]: fix

File: services/agents/log_level.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/version"
+)
+
+// Log level available in exporters with pmm 2.28
+var exporterLogLevelCommandVersion = version.MustParse("2.27.99")
+
+func withLogLevel(args []string, logLevel *string, pmmAgentVersion *version.Parsed) []string {
+	if pointer.GetString(logLevel) != "" && !pmmAgentVersion.Less(exporterLogLevelCommandVersion) {
+		args = append(args, "-log.level="+*logLevel)
```
[the remark this answers is not in the document]
Comment (YaroslavPodorvanov) [reply services/agents/log_level.go]: fix

File: services/agents/log_level.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package agents
+
+import (
+	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/version"
+)
+
+// Log level available in exporters with pmm 2.28
+var exporterLogLevelCommandVersion = version.MustParse("2.28.0")
```
Comment (artemgavrilov) [inline services/agents/log_level.go:9]: I guess it will go to 2.29

File: services/management/mongodb.go [hunk 4]
```diff
@@ -93,6 +93,7 @@ func (s *MongoDBService) Add(ctx context.Context, req *managementpb.AddMongoDBRe
 			MongoDBOptions:    mongoDBOptions,
 			PushMetrics:       isPushMode(req.MetricsMode),
 			DisableCollectors: req.DisableCollectors,
+			LogLevel:          req.LogLevel.String(),
```
Comment (artemgavrilov) [inline services/management/mongodb.go:96]: So we expect that log level always specified? No default value?

Comment (YaroslavPodorvanov) [reply services/management/mongodb.go:96]: Default is warn like in issue https://jira.percona.com/browse/PMM-7326
```
while the default should be `–log–level=WARN`
```

So "req.LogLevel.String()" return "warn" if not set in pmm-admin

File: services/agents/log_level.go (hunk 3 above)
Comment (YaroslavPodorvanov) [reply services/agents/log_level.go:9]: So main dependency is support in exporters side. This support added in 2.28.0 https://jira.percona.com/browse/PMM-9840

**Review by artemgavrilov — APPROVED** [review]
CI is failing

**Pull request merged**